parsers/cobol.c

```diff
@@ -41,33 +49,120 @@ static kindDefinition CobolKinds[] = {
 
 static tagRegexTable cobolTagRegexTable[] = {
 	{ "......\\*.*", "", "", "{exclusive}" },
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "[FSR]D[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
 	 "f,fd,file descriptions (FD, SD, RD)", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "[0-9]+[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
 	 "g,group,group items", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "PROGRAM-ID\\.[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
 	 "P,program,program ids", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "([A-Z0-9][A-Z0-9-]*)[ \t]+SECTION\\.", "\\1",
 	 "s,section,sections", "i"},
-	{ COBOL_REGEX_PREFIX
+	{ COBOL_REGEX_PREFIX_FIXED
 	  "([A-Z0-9][A-Z0-9-]*)[ \t]+DIVISION\\.", "\\1",
 	  "D,division,divisions", "i"},
+	{ COBOL_REGEX_PREFIX_RELAXED
+	"[FSR]D[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
+	"f,fd,file descriptions (FD, SD, RD)", "i" },
+	{ COBOL_REGEX_PREFIX_RELAXED
+	"[0-9]+[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
+	"g,group,group items", "i" },
+	{ COBOL_REGEX_PREFIX_RELAXED
+	"PROGRAM-ID\\.[ \t]+([A-Z0-9][A-Z0-9-]*)\\.", "\\1",
+	"P,program,program ids", "i" },
+	{ COBOL_REGEX_PREFIX_RELAXED
+	"([A-Z0-9][A-Z0-9-]*)[ \t]+SECTION\\.", "\\1",
+	"s,section,sections", "i" },
+	{ COBOL_REGEX_PREFIX_RELAXED
+	"([A-Z0-9][A-Z0-9-]*)[ \t]+DIVISION\\.", "\\1",
+	"D,division,divisions", "i" },
 };
 
 typedef enum {
 	K,
 } cobolKeyword;
 
 static const keywordTable cobolKeywordTable[] = {
-	{ "CONTINUE", K },
-	{ "END-EXEC", K },
-	{ "EXIT", K },
-	{ "FILLER", K },
-};
+	{ "ACCEPT", K },{ "ACCESS", K },{ "ADD", K },{ "ADDRESS", K },{ "ADVANCING", K },{ "AFTER", K },
```

Comment:
Does introducing new keywords change the result of parsing?
If yes, you have to do:
1. make a commit only for introducing keywrods.
   I don't know Cobol well but I guess 
   the  introducing keywords is nothing to do with "sourceformat free/variable".
   In that case, commits should be separated for each purpose of changing.
2. test cases verifying the impact of the introducing keywords.